tests/h/services/search_index/service_test.py

```diff
@@ -223,22 +215,17 @@ def test_we_fallback_to_async_if_sync_fails(
         assert result == async_handler.return_value
 
     @pytest.fixture(autouse=True)
-    def handler_for(self, patch, add_annotation_by_id, delete_annotation_by_id):
-        add_annotation_task = patch("h.services.search_index.service.add_annotation")
-        delete_annotation_task = patch(
-            "h.services.search_index.service.delete_annotation"
-        )
```

Comment:
Having these two patch happen with this fixture is unexpected because it doesn't follow our usual style, and it means that the two patch fixtures are only available to this test class not the rest of the tests. These are patches for module-level imports so our style is for them to have their own module-scoped autouse fixtures and for the names of the fixtures to be the same as the names they patch (no `_task` suffix). I've moved these into the new module-level `index` fixture